ci-operator/step-registry/ipi/gcp/techpreview/ipi-gcp-techpreview-workflow.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "ipi/gcp/techpreview/ipi-gcp-techpreview-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"jianlinliu",
+			"patrickdillon",
+			"technical-release-team-approvers",
+			"vrutkovs",
+			"yunjiang29"
+		]
+	}
+}
```

Comment:
This is copied from https://github.com/openshift/release/blob/02e8f2de79b7a6ca4c6b22d89f27c0a4abe91ddf/ci-operator/step-registry/ipi/gcp/ipi-gcp-workflow.metadata.json.